bids/variables/entities.py

```diff
@@ -84,6 +84,7 @@ class NodeIndex(Node):
     """Represents the top level in a BIDS hierarchy. """
 
     def __init__(self):
+        super().__init__("top_of_bids_heirarchy", None)
```

Comment:
I'm not sure this makes sense. @tyarkoni Should `NodeIndex` itself subclass `Node`? It doesn't seem to use any of the attributes of `Node` that I can see.

Reply:
Yeah see my review comment.

Reply:
pulled off into separate PR.